File: RSA/RSA_decryption.cpp
#include <iostream>
using namespace std;

unsigned long long msg_decrypted =1, i, ans=0;
unsigned long long e =35, p=23, q=53, n = p*q; 
unsigned long long message = 50;

int main(){
    for (i=1; i<=e; i++) {
        msg_decrypted = (msg_decrypted*message)%n; //message^key mod N
    }
    ans = msg_decrypted;
    cout << ans << endl;
    return 0;
}

File: RSA/RSA_decryption.py
e, p, q = 35, 23, 53   #49,21  #101,23  #131,53 
n = int(p) * int(q)
message = 50
msg_decrypted = 1

for i in range (e):                
    msg_decrypted = (msg_decrypted * message) % n

print(msg_decrypted)
